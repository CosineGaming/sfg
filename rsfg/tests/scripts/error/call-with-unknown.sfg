fn main()
	log(unknown_ident)

