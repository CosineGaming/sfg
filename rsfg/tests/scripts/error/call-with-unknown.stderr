[ERROR] referenced unknown identifier unknown_ident at 2:6